Update unplayed search filter usage (#4320)
ppy/osu-wiki: wiki/Interface/en.md

- `mode`: Mode. Value can be `osu`, `taiko`, `catchthebeat`, or `mania`, or `o`/`t`/`c`/`m` for short.
- `status`: Ranked status. Value can be `ranked`, `approved`, `pending`, `notsubmitted`, `unknown`, or `loved`, or `r`/`a`/`p`/`n`/`u`/`l` for short.
- `played`: Time since last played in days

> - `unplayed`: Unplayed maps. No value. Only supported comparison is `=`. Any other comparison will result in the comparison of `=`.

Reviewer: ```suggestion
- `unplayed`: Unplayed maps. Any comparison can be used, as long as it has no set value.
```
might be better

Author: That would imply, that `!=`
works, which is not the case, and that `<`,`<=`,`>` and `>=` give meaningful and different search results.

Reviewer: what about
```suggestion
- `unplayed`: Unplayed maps. Any comparison with no set value can be used, however all of them will have the same result.
```

Author: I feel like you would need to read that twice to understand, because the result mentioned isn't always connected to the Unplayed Maps.

What about:

```suggestion

-- `unplayed`: Unplayed maps. No value. Only supported comparison is `=`. Any other comparison will result in the comparison of `=`.
+- `unplayed`: Shows only Unplayed maps. A comparison with no set value must be used. The comparison itself is ignored.

```
